unity-client/Assets/Scripts/MainScripts/DCL/Components/ParametrizedShapes/BoxShape.cs

```diff
@@ -10,6 +10,8 @@ public class BoxShape : ParametrizedShape<BoxShape.Model>
         [System.Serializable]
         new public class Model : BaseShape.Model
         {
+            public float[] uvs;
```

Comment:
We should add or adapt an existing test for the new uv feature. 

Check this as reference:
https://github.com/decentraland/explorer/blob/16916660ca5b814362e9f70c5c5cd57947709322/unity-client/Assets/Scripts/MainScripts/DCL/Components/ParametrizedShapes/Tests/PlaneShapeShould.cs#L24